sem/manager.py

```diff
@@ -290,7 +292,32 @@ def check_and_fill_parameters(self, param_list, needs_rngrun):
     # Simulation running #
     ######################
 
-    def run_simulations(self, param_list, show_progress=True, stop_on_errors=True):
+    def parse_log_component(self,log_component):
+        #Verifies if the log levels passed in the log_component dictionary are correct 
+        # and removes the level_ prefix (if present) from the log levels and converts 
+        # the levels to corresponding levels without the level_prefix
+        # For example,
+        # 'level_debug' get converted to 'warn|error|debug'
+        ret_dict = {}
+        log_level_list = ['error','warn','debug','info','function','logic','all']
```

Comment:
use standard python coding style
```suggestion
        log_level_list = ['error', 'warn', 'debug', 'info', 'function', 'logic', 'all']
```